src/probabilities.jl

```diff
@@ -140,7 +148,7 @@ function probabilities! end
 """
     outcome_space(est::ProbabilitiesEstimator, x) → Ω
 
-Return a container containing all _possible_ outcomes of `est` for input `x`.
+Return a sorted container containing all _possible_ outcomes of `est` for input `x`.
```

Comment:
By "sorted", we mean the sorting that *we* impose on an estimator `est` by hard-coding it, right? Not the sorting achieved by calling `sort(ospace)`, where `ospace = outcome_space(est, x)`? 

Do we guarantee that the outcome order we define for each estimator is the same as when calling `sort` on the outcome space vector? If so, then this should also be mentioned in the developer docs.